tests/profile/fixtures/great_expectations_titanic_0.13.yml

```diff
@@ -8,7 +8,7 @@
 
 # config_version refers to the syntactic version of this config file, and is used in maintaining backwards compatibility
 # It is auto-generated and usually does not need to be changed.
-config_version: 2.0
```

Comment:
Do we still support config version 2? This change seems reasonable, but hoping we have other tests around 2.0

Reply:
I think not; moving forward we might just want to axe the config version entirely? Might be worth a larger discussion